component/upgrade.jsonnet

```
@@ -63,14 +63,17 @@ local job = kube.Job(name) {
             args: [ '-eu', '-c', upgradeScript ],
             env: [
               { name: 'CRDS_TO_PATCH', value: std.join(' ', crds) },
+              { name: 'HOME', value: '/home/upgrade' },
```

Instead of a 2nd emptyDir it would be perfectly fine to just use `/export` as the home directory in this particular case:

```suggestion
              { name: 'HOME', value: '/export' },
```

---

yep you're right 👍